packages/bulkapi/src/database/LionWebQueries.ts

```diff
@@ -190,7 +184,7 @@ export class LionWebQueries {
         // Now get all children of the orphans
         const orphansContainedChildren = await this.getNodeTree(
             removedAndNotAddedChildren.map(rm => rm.childId),
-            999
+            99999
```

Comment:
Magic number? Could it be configured somewhere, or at least be put into a constant with a descriptive name?

Reply:
Done, called FOREVER